draft-ietf-httpbis-cache-digest.md

```diff
@@ -204,20 +204,21 @@ Given the following inputs:
 4. Let `key` be the return value of {{key}} with `URL` and `ETag` as inputs.
 5. Let `h1` be the return value of {{hash}} with `key` and `N` as inputs.
 6. Let `fingerprint` be the return value of {{fingerprint}} with `key`, `N` and `f` as inputs.
-7. Let `h2` be the return value of {{hash}} with `fingerprint` and `N` as inputs, XORed with `h1`.
-8. Let `h` be either `h1` or `h2`, picked in random.
-9. Let `position_start` be 40 + `h` * `f`.
-10. Let `position_end` be `position_start` + `f` \* `b`.
-11. While `position_start` < `position_end`:
+7. Let `fingerprint-string` be the value of `fingerprint` in base 10, expressed as a string.
```

Comment:
@yoavweiss Curious... May I ask why this change? I don't see any issues with it. Just don't understand what it means.

Reply:
It defines a way to convert fingerprint into a string, so that we can apply {{hash}} on it